proto/api/v2/compliance_scan_schedule_service.proto

```diff
@@ -0,0 +1,110 @@
+syntax = "proto3";
+
+option go_package = "v2";
+option java_package = "io.stackrox.proto.api.v2";
+
+import "google/protobuf/timestamp.proto";
+import weak "google/api/annotations.proto";
+import "api/v2/common.proto";
+import "api/v2/user.proto";
+
+package v2;
+
+// Represents the role of the node within the cluster
+enum NodeRole {
+    INFRA = 0;
+    WORKER = 1;
+    MASTER = 2;
+}
+
+// ClusterScanStatus holds status based on cluster in the event that a schedule was
+// successfully applied to some clusters but not others.
+// Next available tag: 3
+message ClusterScanStatus {
+    string                    cluster_id = 1;
+    repeated string           errors     = 2;
+}
+
+// Next available tag: 7
+message BaseComplianceScanScheduleSettings {
+    bool                  auto_apply_remediations  = 1;
+    bool                  auto_update_remediations = 2;
```

Comment:
We won't have support for this for a couple of releases at the least, so I recommend dropping the two fields.